uniref-rest/src/main/java/org/uniprot/api/uniref/repository/store/UniRefStreamConfig.java

```diff
@@ -63,13 +64,15 @@ public StoreStreamer<UniRefEntryLight> unirefEntryStoreStreamer(
                         .withDelay(Duration.ofMillis(streamConfig.getStoreFetchRetryDelayMillis()))
                         .withMaxRetries(streamConfig.getStoreFetchMaxRetries());
 
-        return StoreStreamer.<UniRefEntryLight>builder()
-                .streamConfig(streamConfig)
-                .storeClient(uniRefLightStoreClient)
-                .tupleStreamTemplate(tupleStreamTemplate)
-                .storeFetchRetryPolicy(storeRetryPolicy)
-                .documentIdStream(documentIdStream)
-                .build();
+        StoreStreamerConfig<UniRefEntryLight> storeStreamerConfig =
```

Comment:
i wonder why in variable?

Reply:
I am sorry, I did not understand your question.